guides/common/modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc

```diff
@@ -6,3 +6,8 @@
 You can restore {ProjectServer} or {SmartProxyServer} from the backup data that you create as part of xref:backing-up-{project-context}-server-and-{smart-proxy-context}_{context}[].
 This process outlines how to restore the backup on the same server that generated the backup, and all data covered by the backup is deleted on the target system.
 If the original system is unavailable, provision a system with the same configuration settings and host name.
++
```

Comment:
This renders badly?
<img width="1552" height="513" alt="image" src="https://github.com/user-attachments/assets/3bc10b8c-c333-4961-bd36-cd2067f97c94" />